test/extensions/filters/network/mysql_proxy/BUILD

```diff
@@ -0,0 +1,73 @@
+licenses(["notice"])  # Apache 2
+
+load(
+    "//bazel:envoy_build_system.bzl",
+    "envoy_cc_test_library",
+    "envoy_package",
+)
+load(
+    "//test/extensions:extensions_build_system.bzl",
+    "envoy_extension_cc_test",
+)
+
+envoy_package()
+
+envoy_cc_test_library(
+    name = "mysql_test_utils_lib",
+    srcs = ["mysql_test_utils.cc"],
+    hdrs = ["mysql_test_utils.h"],
+    deps = [
+        "//source/extensions/filters/network/mysql_proxy:config",
+        "//test/integration:integration_lib",
```

Comment:
We can remove this if: https://github.com/envoyproxy/envoy/pull/4975/files#r233189621 is resolved?